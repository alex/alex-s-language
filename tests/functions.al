def fact(n):
    if n == 2 or n == 1:
        return n
    return n * fact(n)

print(fact(3))
