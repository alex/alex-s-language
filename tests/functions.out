2
6
5040
13
